vulyk/blueprints/gamification/models/rules.py

```diff
@@ -101,7 +101,7 @@ class RuleModel(Document):
     """
     Database-specific rule representation
     """
-    id = IntField(required=True, primary_key=True)
+    id = StringField(required=True, primary_key=True)
```

Comment:
One more thing. Field `Rule._hash` and `ProjectRule._hash`, as well as their `id` projections have `int` type annotations. Ditto for constructor methods description.

Reply:
Done.